bricks/feature_brick_plus/brick.yaml

```
@@ -0,0 +1,32 @@
+
+vars:
+  feature_name:
+    type: string
+    default: login
+    description: The feature name
+    prompt: What is the feature name?
+  feature_type:
+    type: enum
+    default: conventional
+    description: The feature type or general structure
+    prompt: What kind of feature are you building? 
```

yamllint

[error] trailing spaces

(trailing-spaces)